coalib/results/Diff.py

```diff
@@ -432,7 +451,7 @@ def __bool__(self):
         """
         return (self.rename is not False or
                 self.delete is True or
-                self.modified != self._file)
+                self._raw_modified() != self._file)
```

Comment:
consider this:
```
d = Diff(['1', '2', '3'])
d.modify_line(1, '1\n')
```
This would effectively render no difference in `d.modified` before and after the change, but internally we say that something changed because we handle things without a \n.